store/sqlstore/post_store.go

```diff
@@ -1780,3 +1790,34 @@ func (s *SqlPostStore) SearchPostsInTeamForUser(paramsList []*model.SearchParams
 
 	return model.MakePostSearchResults(posts, nil), nil
 }
+
+func wrapEmails(terms string) (string, bool) {
```

Comment:
Probably we want to rename this to something like `quoteEmailTerms`.

Also, I'd move this to [here](https://github.com/mattermost/mattermost-server/blob/master/store/sqlstore/utils.go)